index.html

```diff
@@ -352,40 +476,40 @@ <h4 class="featureText">Terminal</h4>
         </ul>
       </section>
       <section class="flex-container" id="formContainer">
-        <p id="contactDescription">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-        <form class="flex-container" action="https://formspree.io/f/xnqyndrw" method="POST">
+        <p id="contactDescription">I'm always interested <br>in hearing about new <br>projects, so if you'd <br>like to chat please <br>get in touch.</p>
+        <form class="flex-container" id="contactForm" action="https://formspree.io/f/xnqyndrw" method="POST">
           <input class="formElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
           <input class="formElement" type="email" name="email" placeholder="Email address" required>
           <textarea class="formElement name="message" placeholder="Enter text here" maxlength="500" required></textarea>
           <button class="button" id="submitForm" type="submit">Get in touch</button>
         </form>
       </section>
     </div>
```

Comment:
[REQUIRED]As requested by the previous reviewer, I think your project still looks a bit different from the design😉 don't forget it is a requirement that we stick to the design even as we make our page responsive😉 kindly see the guiding lines on the image and make the changes🤗

YOUR APP
<img width="946" alt="your___________" src="https://user-images.githubusercontent.com/64643995/213652352-adb8b329-5bae-42f7-bc7d-f310fcf6e647.png">

DESIGN
<img width="364" alt="design_________" src="https://user-images.githubusercontent.com/64643995/213652394-8b54c23e-c8ef-4522-872b-3ba552c3ff5a.png">

[OPTIONAL] I think the thick black line highlighted with yellow shouldn't be a part of your application😉 That is just a navigation usually present in mobile phones😉 probably the designer uses that to have a better illustration of the look of the application on mobile devices😉 I suggest you take it away as it makes your app look less professional😉